docs/mkdocs/docs/api/basic_json/operator_ltlt.md

```diff
@@ -1,62 +1,86 @@
+      result is the same as calling `dump(4)`.
+    - The indentation character can be controlled with the member variable `fill` of the output stream `o`.
+      For instance, the manipulator `std::setfill('\\t')` sets indentation to use a tab character rather than the
+      default space character.
+1. Write a string representation of the given JSON pointer `ptr` to the output stream `o`. The string representation is
+   obtained using the [`to_string`](../json_pointer/to_string.md) member function.
 
 ## Parameters
 
 `o` (in, out)
-:   stream to serialize to
+:   stream to write to
 
 `j` (in)
 :   JSON value to serialize
 
+`ptr` (in)
+:   JSON pointer to write
+
 ## Return value
 
 the stream `o`
 
 ## Exceptions
 
-Throws [`type_error.316`](../../home/exceptions.md#jsonexceptiontype_error316) if a string stored inside the JSON value
-is not UTF-8 encoded. Note that unlike the [`dump`](dump.md) member functions, no `error_handler` can be set.
+1. Throws [`type_error.316`](../../home/exceptions.md#jsonexceptiontype_error316) if a string stored inside the JSON
+   value is not UTF-8 encoded. Note that unlike the [`dump`](dump.md) member functions, no `error_handler` can be set.
+2. None.
 
 ## Complexity
 
 Linear.
 
+
+## Notes
+
+!!! warning "Deprecation"
+
+    Function  `#!cpp std::ostream& operator<<(std::ostream& o, const basic_json& j)` replaces function
+    `#!cpp std::ostream& operator>>(const basic_json& j, std::ostream& o)` which has been deprecated in version 3.0.0.
+    It will be removed in version 4.0.0. Please replace calls like `#!cpp j >> o;` with `#!cpp o << j;`.
+
 ## Examples
 
-??? example
+??? example "Example: (1) serialize JSON value to stream"
 
     The example below shows the serialization with different parameters to `width` to adjust the indentation level.
-        
+    
     ```cpp
-    --8<-- "examples/operator_serialize.cpp"
+    --8<-- "examples/operator_ltlt__basic_json.cpp"
     ```
     
     Output:
     
     ```json
-    --8<-- "examples/operator_serialize.output"
+    --8<-- "examples/operator_ltlt__basic_json.output"
     ```
 
-## Version history
-
-- Added in version 1.0.0
-- Support for indentation character added in version 3.0.0.
+??? example "Example: (2) write JSON pointer to stream"
 
-!!! warning "Deprecation"
+    The example below shows how to write a JSON pointer to a stream.
+    
+    ```cpp
+    --8<-- "examples/operator_ltlt__json_pointer.cpp"
+    ```
+    
+    Output:
+    
+    ```json
+    --8<-- "examples/operator_ltlt__json_pointer.output"
+    ```
+## Version history
 
-    This function replaces function `#!cpp std::ostream& operator>>(const basic_json& j, std::ostream& o)` which has
-    been deprecated in version 3.0.0. It will be removed in version 4.0.0. Please replace calls like `#!cpp j >> o;`
-    with `#!cpp o << j;`.
+1. Added in version 1.0.0. Support for indentation character added in version 3.0.0.
+2. Added in version 3.11.0.
```

Comment:
We usually have deprecation notices in the version history.

Reply:
Hm. I've not done so in my PRs, which is why this looked out of place.

See:
https://json.nlohmann.me/api/json_pointer/#template-parameters
`api/macros/json_use_legacy_discarded_value_comparison.md`

This makes more sense to me. :man_shrugging:

Reply:
I don't say that such a section is not helpful. I just would like to also keep it in the version history.

Reply:
Let me know if my revision is satisfactory.